app/controllers/concerns/proxyable.rb

```diff
@@ -4,9 +4,9 @@ module Proxyable
   def call
     self.response_body = proxy_responce.body
     self.status = proxy_responce.status
-    response.headers = {
+    response.headers = response.headers.merge({
       'Content-Type' => proxy_responce.headers['content-type'].join(',')
-    }
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.